src/java.base/share/classes/javax/crypto/CryptoPermission.java

```diff
@@ -35,9 +35,9 @@
 import javax.crypto.spec.*;
 
 /**
- * The CryptoPermission class extends the
- * java.security.Permission class. A
- * CryptoPermission object is used to represent
+ * The <code>CryptoPermission</code> class extends the
+ * <code>java.security.Permission</code> class. A
+ * <code>CryptoPermission</code> object is used to represent
```

Comment:
Use {@code } for all of the javadoc in this class?

Reply:
using `{@code }`